public/data/itself13.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Ayush Singhal",
+    "bio": "MERN stack developer",
+    "avatar": "https://github.com/itself13.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/itself13",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dx_singhal",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Instagram",
+        "url": "https://www.instagram.com/ii_am_ayuuu/",
+        "icon": "Instagram"
+      }
+    ]
+  }
```

Comment:
```suggestion
  }
  
```
### Just add a single line below 22 (With no in Spaces in that).

Reply:
This is a wrong suggestion @Pradumnasaraf, please check my comments below...

Reply:
How this is wrong? Can you explain?

Reply:
@krishguptadev How this is wrong? Can you explain?